docs/getting_started/install_manually.md

```diff
@@ -1,29 +1,284 @@
+
+    ``` bash
+    fallocate -l 4G /swapfile
+    chmod 600 /swapfile
+    mkswap /swapfile
+    swapon /swapfile
+    echo "/swapfile   none    swap    sw    0   0" >> /etc/fstab
+    sysctl vm.swappiness=10
+    echo "vm.swappiness=10" >> /etc/sysctl.conf
+    sysctl vm.vfs_cache_pressure=50
+    echo "vm.vfs_cache_pressure=50" >> /etc/sysctl.conf
+    ```
+
+## 2. Get Composer
+
+a. Install Composer, the PHP command line dependency manager, by running the following command in the terminal:
+
+``` bash
+php -r "readfile('https://getcomposer.org/installer');" | php
+```
+
+b. Move the downloaded `composer.phar` file to a globally-available path:
+
+``` bash
+mv composer.phar /usr/local/bin/composer
+```
+
+## 3. Download eZ Platform
+
+You can download eZ Platform in two ways: by downloading an archive, or cloning the GitHub repository:
+
+a. Download an archive
+
+- If you are installing eZ Platform, download the latest archive from [ezplatform.com](https://ezplatform.com/#download-option).
+- For licensed eZ Enterprise customers, download your file from the [Support portal](https://support.ez.no/Downloads).
+
+Extract the archive into `/var/www/ezplatform`.
+
+b. Clone GitHub repository
+
+You can also clone one of the [repositories from GitHub](#available-distributions).
+
+``` bash
+git clone https://github.com/ezsystems/ezplatform.git /var/www/ezplatform
+```
+
+You can check out a tag, or use the `master` branch if you are interested in working with the latest version.
+
+!!! tip
+
+    You can use any other folder name in place of `ezplatform` in the examples above.
+    You'll point your virtual host to this folder to use as its root.
+
+## 4. Create a database
+
+Create a new database (you can substitute `ezplatform` with the database name you want to use, but keep it in mind in next steps):
+
+``` bash
+mysql -u root -e 'CREATE DATABASE ezplatform CHARACTER SET utf8;'
+```
+
+## 5. Run installation scripts
+
+Composer will look inside the `composer.json` file and install all of the packages required to run eZ Platform.
+The `app/console` script will then install eZ Platform for your desired environment (dev/prod).
+
+### a. Run composer install
+
+From the folder into which you downloaded eZ Platform, run:
+
+``` bash
+php -d memory_limit=-1 composer install
+```
+
+Once the installer gets to the point that it creates `app/config/parameters.yml`, you will be presented with a few decisions:
+
+1. Choose a [secret](http://symfony.com/doc/current/reference/configuration/framework.html#secret); choose any random string you like, made up of characters, numbers, and symbols, up to around 32 characters. This is used by Symfony when generating [CSRF tokens](http://symfony.com/doc/current/book/forms.html#forms-csrf), [encrypting cookies](http://symfony.com/doc/current/cookbook/security/remember_me.html), and for creating signed URIs when using [ESI (Edge Side Includes)](http://symfony.com/doc/current/book/http_cache.html#edge-side-includes).
```

Comment:
To be easier to follow: Choose a [secret](http://symfony.com/doc/current/reference/configuration/framework.html#secret) it should be/it is a random string, made up of up to 32 characters, numbers, and symbols.